myuw/templates/handlebars/card/schedule/visual.html

```diff
@@ -4,155 +4,159 @@
-    <div class="visual-schedule {{schedule_hours_class}}" aria-hidden="true">
-	<div class="time-bar day-col2 hours-{{total_hours}} clearfix ">
-	    {{#each display_hours}}
-	    <div class="hour-label" style="position:absolute;top:{{time_percentage position ../start_time ../end_time }}%;"><p>{{format_schedule_hour hour hour_count}}</p>
-	    </div>
-	    {{/each}}
-	</div>
-
-	<div {{#if has_6_days}}class="six-day"{{else}}class="five-day"{{/if}}>
-	    <div class="day-label">MON</div>
-	    <div class="day-col hours-{{total_hours}}">
-	        {{ show_card_days_meetings monday start_time end_time }}
-	    </div>
-	</div>
-
-	<div {{#if has_6_days}}class="six-day"{{else}}class="five-day"{{/if}}>
-	    <div class="day-label">TUE</div>
-	    <div class="day-col hours-{{total_hours}}">
-	        {{ show_card_days_meetings tuesday start_time end_time }}
-	    </div>
-	</div>
-
-	<div {{#if has_6_days}}class="six-day"{{else}}class="five-day"{{/if}}>
-	    <div class="day-label">WED</div>
-	    <div class="day-col hours-{{total_hours}}">
-	        {{ show_card_days_meetings wednesday start_time end_time }}
-	    </div>
-	</div>
-
-	<div {{#if has_6_days}}class="six-day"{{else}}class="five-day"{{/if}}>
-	    <div class="day-label">THU</div>
-	    <div class="day-col hours-{{total_hours}}">
-	        {{ show_card_days_meetings thursday start_time end_time }}
-	    </div>
-	</div>
-
-	<div {{#if has_6_days}}class="six-day"{{else}}class="five-day"{{/if}}>
-	    <div class="day-label">FRI</div>
-	    <div class="day-col hours-{{total_hours}}">
-	        {{ show_card_days_meetings friday start_time end_time }}
-	    </div>
-	</div>
-
-	{{#if has_6_days}}
-	<div class="six-day">
-	    <div class="day-label">SAT</div>
-	    <div class="day-col hours-{{total_hours}}">
-	        {{ show_card_days_meetings saturday start_time end_time }}
-	    </div>
-	</div>
-	{{/if}}
-    </div>
-    {{/if}}
-    <!-- end week view -->
-
-    <!-- begin - if there are meetings with times to be arranged -->
-    {{#if courses_meeting_tbd }}
-    <div class="courseTBD" aria-hidden="true">
-        {{#each courses_meeting_tbd }}
-	    <div class="pull-left course-box">
-	        <div class="c{{color_id}} course-info"><a href="/{{#if ../is_instructor}}teaching{{else}}academics{{/if}}/#{{curriculum}}-{{course_number}}-{{section_id}}" style="color: white;" class="show_section_card">{{curriculum}} {{course_number}}&nbsp;{{section_id}}</a></div>
-	        <div style="font-size:.8em; color:#555; text-align:center; margin-top:3px;">Room TBD</div>
-	    </div>
-        {{/each}}
-    </div>
-    {{/if}}
-    <!-- end - if there are meetings with times to be arranged -->
 
+    <h3 id="quarter-info" >
+            {{ucfirst quarter}} {{year}} Schedule
+            {{#if summer_term}}
+            <span class="shortTitle">{{summer_term}}</span>
+            {{/if}}
+    </h3>
+    <div><p class="sr-only"><a href="#CourseCards">Skip the visual schedule and go to course cards</a></p></div>
```

Comment:
This jump link location is no longer valid: `<a href="#CourseCards">` I think the correct location is: "/academics#academics_content_cards"

That being said, I'm not sure this is all that helpful though. The course cards are on a completely separate page which can be easily reached from the global nav.

Reply:
A good catch! 
Understood your point on the usefulness. Until we fixe the Visual schedule table, it can crutch users to the course schedule info,  IMO.